configure.ac

```diff
@@ -768,6 +771,39 @@ AS_IF([test x$with_lustre != xno],[
        MOTR_LINUX_SENDFILE
 ])
 
+#
+# Checking libfabric library----------------------------------------------- {{{1
+#
+
+AC_ARG_WITH([libfab],
+        [AS_HELP_STRING([--with-libfab=path],
+                        [set path to user installed libfabric headers and library])],
+        [LIBFABRIC=$with_libfab],
+        [LIBFABRIC=""]
+)
+
+AS_IF([test -z "$with_libfab" && ! test -f "/usr/include/rdma/fabric.h"],
+      [AC_MSG_ERROR([Libfabric cannot be found! Please, install \
+libfabric-devel package or provide the path to a directory with built \
+libfabric sources, using --with-libfab option.])]
+)
+
+AS_IF([test -z $with_libfab],[
+        LIB_FAB="-lfabric"
+      ],[
+        LIB_FAB_INCLUDES="-I$LIBFABRIC/include"
+        LIB_FAB="-L$LIBFABRIC/lib -lfabric"
+      ]
+)
+
+AC_DEFINE([ENABLE_LIB_FAB])
+AC_SUBST([LIB_FAB])
+AC_SUBST([LIB_FAB_INCLUDES])
+
+AS_IF([! test -z "$with_libfab"],
+        [ M0_CPPFLAGS="$M0_CPPFLAGS $LIB_FAB_INCLUDES"]
+)
+
```

Comment:
Recently, "libedit" is added as a dependency for hsm.
So, this patch need something similar:
```
apr-u520428l001:cortx-motr huanghua$ git grep libedit
configure.ac:# check for libedit library
configure.ac:        [libedit cannot be found! Try to install libedit-devel.]
cortx-motr.spec.in:BuildRequires:  libedit-devel
cortx-motr.spec.in:Requires:       libedit
scripts/provisioning/roles/motr-build/vars/RedHat.yml:  - libedit-devel
scripts/provisioning/roles/motr-runtime/vars/RedHat.yml:  - libedit

```